presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -206,7 +206,7 @@ public int getMaxSplitsPerSecond()
     @ConfigDescription("Throttles the maximum number of splits that can be assigned to tasks per second. 0 disables throttling.")
```

Comment:
I think the "0 disables" is out of date now. Probably just remove it.

Reply:
Yes, thanks